### ISSUE TITLE
GWT hidden iframe violates a11y guidelines

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#4887]
### System details

    RStudio Edition : Desktop or Server
    RStudio Version : 1.3.215
    OS Version      : any
    R Version       : any

### Steps to reproduce the problem
Examine IDE HTML and/or run an automated a11y testing tool such as Lighthouse in Chrome.

![2019-06-03_13-07-14](https://user-images.githubusercontent.com/10569626/58831047-989c1a80-8600-11e9-94a2-72268eff742b.png)

### Describe the problem in detail
Critical error (WCAG 2.1 level A violation) flagged related to the hidden iframe that GWT uses for loading scripts. iframes MUST have a title.

https://dequeuniversity.com/rules/axe/3.2/frame-title
https://dequeuniversity.com/tips/provide-iframe-titles

### Describe the behavior you expected
Don't have this violation. See also: https://github.com/gwtproject/gwt/issues/9443
